### DISCUSSION TITLE
Welcome to STranslate Discussions!

### DISCUSSION THREAD

**Discussion by ZGGSONG** [STranslate/STranslate#46] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZGGSONG** [comment]
![image](https://github.com/ZGGSONG/STranslate/assets/49741009/ceb6dee3-d2fd-41d0-9e18-d2d08f02b7e6)

**Comment by chunme** [comment]
win10 LTSC2019 打不开

**Reply by ZGGSONG** [reply]
不是最新版就去升级
https://github.com/STranslate/STranslate/releases

**Reply by chunme** [reply]
感谢回复，是最新205 点击无反应
